.github/workflows/tests.yml

```diff
@@ -0,0 +1,40 @@
+name: Open Source Programs Run Tests
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    name: Run tests
+    runs-on: ubuntu-latest
+    services:
+      postgres:
+        image: postgres
+        env:
+          POSTGRES_PASSWORD: osp
+        options: >-
+          --health-cmd pg_isready
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+        ports:
+          - 5432:5432
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python 3.7
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.7
+    - name: Installing dependencies
+      run: |
+        python -m pip install --upgrade pip
+        if [ -f requirements.txt ]; then pip install -r requirements.txt; fi
+    - name: Creating Zulip API key
+      run: |
+        echo "[api]" >> download
+        echo "email=sankalp123427@gmail.com" >> download
+        echo "key=8j7pGWLMuFdb7maOyv0uTTg0nhkQE7Nc" >> download
```

Comment:
oh no, please remove this. Can you do a little search, we can have environment variables set for GitHub Actions, so you don't need to expose these values. Please squash the commits so this gets taken out

Reply:
OK, @isabelcosta

Reply:
here's a little guide about that: https://docs.github.com/en/actions/reference/environment-variables

Reply:
@isabelcosta better to delete this comment too

Reply:
Yes, I have removed it from commit. It would be better if you delete this comment.

Reply:
No problem @codesankalp , now you know for next time ;) ! I will delete,